cookbooks/vectortile/templates/default/import-planet.erb

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Usage
+# sudo -u tileupdate import osmfile.osm.pbf
+# sudo -u tileupdate import --cache 50000 osmfile.osm.pbf
+# This script sets the appropriate Lua and style paths for the osm2pgsql import
+
+set -e
+
+export LUA_PATH='/srv/vector.openstreetmap.org/osm2pgsql-themepark/lua/?.lua;/srv/vector.openstreetmap.org/spirit/?.lua;;'
+
+# Import the osm2pgsql file specified as an argument, using the locations for spirit
+osm2pgsql \
+  --output flex \
+  --style '/srv/vector.openstreetmap.org/spirit/spirit.lua' \
+  --slim \
+  --flat-nodes '/srv/vector.openstreetmap.org/data/nodes.bin' \
+  -d spirit \
+  $@
+
+# Set up replication. This doesn't specify the replication server, so it will use planet.osm.org on extracts
+osm2pgsql-replication init \
+  -d spirit \
+  --server '<%= node[:vectortile][:database][:tag_transform_script] %>'
```

Comment:
That looks like the wrong attribute? Also the comment is out of date now we're specifiying the server.

Reply:
It is the wrong attribute, I'll fix in a subsequent commit